content/develop/data-types/json/_index.md

```diff
@@ -30,107 +30,107 @@ The JSON capability of Redis Stack provides JavaScript Object Notation (JSON) su
-> JSON.NUMINCRBY num $ 1.5
+> JSON.NUMINCRBY crashes $ 1.5
 "[2.5]"
-> JSON.NUMINCRBY num $ -0.75
+> JSON.NUMINCRBY crashes $ -0.75
 "[1.75]"
-> JSON.NUMMULTBY num $ 24
+> JSON.NUMMULTBY crashes $ 24
 "[42]"
-```
+{{< /clients-example >}}
 
 Here's a more interesting example that includes JSON arrays and objects:
 
-```
-> JSON.SET example $ '[ true, { "answer": 42 }, null ]'
+{{< clients-example json_tutorial arr >}}
+> JSON.SET newbike $ '["Deimos", {"crashes": 0}, null]'
 OK
-> JSON.GET example $
-"[[true,{\"answer\":42},null]]"
-> JSON.GET example $[1].answer
-"[42]"
-> JSON.DEL example $[-1]
+> JSON.GET newbike $
+"[[\"Deimos\",{\"crashes\":0},null]]"
+> JSON.GET newbike $[1].crashes
+"[0]"
+> JSON.DEL newbike $[-1]
 (integer) 1
-> JSON.GET example $
-"[[true,{\"answer\":42}]]"
-```
+> JSON.GET newbike $
+"[[\"Deimos\",{\"crashes\":0}]]"
+{{< /clients-example >}}
 
 The [`JSON.DEL`]({{< baseurl >}}/commands/json.del/) command deletes any JSON value you specify with the `path` parameter.
 
 You can manipulate arrays with a dedicated subset of JSON commands:
 
-```
-> JSON.SET arr $ []
+{{< clients-example json_tutorial arr2 >}}
+> JSON.SET riders $ []
 OK
-> JSON.ARRAPPEND arr $ 0
+> JSON.ARRAPPEND riders $ '"Norem"'
 1) (integer) 1
-> JSON.GET arr $
-"[[0]]"
-> JSON.ARRINSERT arr $ 0 -2 -1
-1) (integer) 3
-> JSON.GET arr $
-"[[-2,-1,0]]"
-> JSON.ARRTRIM arr $ 1 1
+> JSON.GET riders $
+"[[\"Norem\"]]"
+> JSON.ARRINSERT riders $ 1 '"Prickett"' '"Royce"' '"Castilla"'
+1) (integer) 4
+> JSON.GET riders $
+"[[\"Norem\",\"Prickett\",\"Royce\",\"Castilla\"]]"
+> JSON.ARRTRIM riders $ 1 1
 1) (integer) 1
-> JSON.GET arr $
-"[[-1]]"
-> JSON.ARRPOP arr $
-1) "-1"
-> JSON.ARRPOP arr $
+> JSON.GET riders $
+"[[\"Prickett\"]]"
+> JSON.ARRPOP riders $
+1) "\"Prickett\""
+> JSON.ARRPOP riders $
 1) (nil)
-```
+{{< /clients-example >}}
 
 JSON objects also have their own commands:
 
-```
-> JSON.SET obj $ '{"name":"Leonard Cohen","lastSeen":1478476800,"loggedOut": true}'
+{{< clients-example json_tutorial obj >}}
+> JSON.SET bike:1 $ '{"model": "Deimos", "brand": "Ergonom", "price": 4972}'
 OK
-> JSON.OBJLEN obj $
+> JSON.OBJLEN bike:1 $
 1) (integer) 3
-> JSON.OBJKEYS obj $
-1) 1) "name"
-   2) "lastSeen"
-   3) "loggedOut"
-```
+> JSON.OBJKEYS bike:1 $
+1) 1) "model"
+   2) "brand"
+   3) "price"
+{{< /clients-example >}}
+
+## Format CLI output
 
-To return a JSON response in a more human-readable format, run `redis-cli` in raw output mode and include formatting keywords such as `INDENT`, `NEWLINE`, and `SPACE` with the [`JSON.GET`]({{< baseurl >}}/commands/json.get/) command:
+The CLI has a raw output mode that lets you add formatting to the output from
+[`JSON.GET`]({{< baseurl >}}/commands/json.get/) to make
+it more readable. To use this, run `redis-cli` with the `--raw` option
+and include formatting keywords such as `INDENT`, `NEWLINE`, and `SPACE`
+with [`JSON.GET`]({{< baseurl >}}/commands/json.get/):
 
-```sh
+{{< clients-example cli only >}}
```

Comment:
Just curious: what was the impetus for this change? If an example is clearly CLI only, as this one is, I don't think it makes sense to use the clients-example shortcode. Change my mind. :)

Reply:
:-D I don't know if it will change your mind or not but I thought it was helpful to see the legend "Redis CLI" explicitly at the top given that most samples in the page are for both CLI and other languages. I can remove it if it's out of step with what we do elsewhere.